cmd/find.go

```diff
@@ -191,31 +200,31 @@ func doFind(clnt Client, ctx *cli.Context) {
 
 		fileContent := parseContent(content)
 		filePath := fileContent.Key
-
-		// traversing in a object store not a file path
-		if err == nil {
-			filePath = path.Join(alias, filePath)
+		if targetAlias != "" {
+			filePath = path.Join(targetAlias, fileContent.Key)
+			fileContent.Key = filePath
 		}
 
 		if ctx.String("maxdepth") != "" {
 			i, e := strconv.Atoi(ctx.String("maxdepth"))
-			s := ""
-
 			fatalIf(probe.NewError(e), "Error parsing string passed to flag maxdepth")
 
-			// we are going to be parsing the path by x amounts
-			pathParts := strings.SplitAfter(filePath, "/")
+			s := ""
+
+			// We are going to be parsing the path by x amounts
+			pathParts := strings.SplitAfter(filePath, separator)
 
-			// handle invalid params
-			// ex. user specifies:
-			// maxdepth 2, but the given object only has a maxdepth of 1
+			// Handle invalid params eg: user specifies
+			// Maxdepth 2, but the given object only
```

Comment:
Is the comment correct, here? given object having maxdepth less than 2 in this case is valid right?

Reply:
It can be more clear,.. let me add.